compute/rn/release-information/release-notes-30-01-update1.adoc

```diff
@@ -170,6 +174,12 @@ Ends the support for the following TLS ciphers for WAAS:
 * TLS_RSA_WITH_AES_128_CBC_SHA 
 * TLS_RSA_WITH_AES_256_CBC_SHA
 
+==== AWS Phase 1 Deprecation of.NET Core 3.1
+
+AWS Lambda runtimes entered .NET Core 3.1 into phase 1 of deprecation on April 3 2023.
+
+Phase 1 deprecation means that AWS Lambda no longer applies security patches or other updates to the runtime. You can no longer create functions that use the runtime, but you can continue to update existing functions. This includes updating the runtime, and rolling back to the previous runtime. Note that functions that use a deprecated runtime are no longer eligible for technical support. See the https://docs.aws.amazon.com/lambda/latest/dg/lambda-runtimes.html[AWS Lamba runtimes documentation] for more details.
```

Comment:
Can the title indicate that it is an AWS announcement?

AWS Announcement on Phase 1 Deprecation of.NET Core 3.1

@rcaballeromx , I think we can make this briefer and link instead. Maybe something like this?

AWS Lambda runtimes entered .NET Core 3.1 into phase 1 of deprecation on April 3 2023, and Prisma Cloud has updated the supported AWS Lambda runtimes in the system requirements.

See the https://docs.aws.amazon.com/lambda/latest/dg/lambda-runtimes.html[AWS Lamba runtimes documentation] for more details.

Reply:
Done